### DISCUSSION TITLE
Energy Dashboard Setup

### DISCUSSION THREAD

**Discussion by Dynamic5912** [BottlecapDave/HomeAssistant-OctopusEnergy#374] [General]
What's the difference between using "Current Consumption" and "Current Accumulative Consumption" for Electric or Gas in the Energy Dashboard?

Does it produce/show results differently?

**Comment by BottlecapDave** [comment]
Hi there.

The results should be identical when it comes to those two sensors. Current consumption is the old recommended way. I'm thinking of deprecating the sensor, but some people rely on the legacy data that comes with the sensor. However I'll delay this until a future release or when the sensor becomes a pain to maintain.

**Comment by markhss** [comment]
Hi, I am not sure if this is answered somewhere else

I would like to use the electricity sensor for current demand to show the live data for the power I am currently using.
I can add that as a card on my dashboard but I can't add it as a grid consumption in the energy dashboard

I have a home mini with octopus and I get live data on the app and on the card I created using the current demand sensor

Is there something I am missing (if it is how do i set it up) or is it a feature that does not exist yet?

**Reply by BottlecapDave** [reply]
Demand is not supported in the energy dashboard as the data reported by OE (watts) is incompatible with the dashboard (kilowatt hours). The current consumption sensors can be added to the energy dashboard as per the [docs](https://github.com/BottlecapDave/HomeAssistant-OctopusEnergy/blob/develop/_docs/energy_dashboard.md#current-consumption)
